README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React ToDo List
 
-## Available Scripts
+> This is a simple ToDo App created using React meant to consolidate all my knowledge about components, props, state, and hooks.   
 
-In the project directory, you can run:
+![Preview](./preview.gif))
```

Comment:
The gif you're using is from the example repo. 
Kindly replace with a snapshot of your project, in order to follow the plagiarism guidelines 😛

Reply:
Hey @eduardosancho. I recorded the gif myself using Peek, but since the styles are the same as the tutorial it looks similar. Additionally, I included a link to the tutorial in the acknowledgment section of my README.md file.

So no worries about plagiarism 😸